File: angular/src/shared/Dto/Client-model.ts
export class ClientModel implements IClientModel {
  Id: number | undefined;
  Code: string | undefined;
  Name: string | undefined;
  Description: string | undefined;
  PhoneNumber: string | undefined;
  MobileNumber: string | undefined;
  Website: string | undefined;

  Email: string | undefined;
  OrganizationId: number | undefined;
  ContactPerson: string | undefined;
  // PersonContactNumber : string | undefined;
  Address1: string | undefined;
  Address2: string | undefined;
  CountryId: number | undefined;
  CityId: number | undefined;
  StateId: number | undefined;
  PostalCode: string | undefined;
  PrefixId: number | undefined;
  Position: string | undefined;
  Date: string | undefined;
  IsDeleted: boolean | undefined;
  IsActive: boolean | undefined;
  Multisite: boolean | undefined;
  LastModifierUserId: number | undefined;
  CreatorUserId: number | undefined;
  RiskId: number | undefined;
  EacodeId: number | undefined;
  NaceCodeId: number | undefined;
  FormName: string | undefined;
  // File: File | undefined;


  // Reviewer : number | undefined;
  // DocumentTypeId : number | undefined;
  // CertificationId : number| undefined ;
  // StatusId : number| undefined ;
  // ModuleId : number | undefined;
  // IsActive : boolean | undefined;
  // IsDeleted:boolean | undefined;
  // File :File | undefined;

  constructor(data?: IClientModel) {
    if (data) {
      for (var property in data) {
        if (data.hasOwnProperty(property))
          (<any>this)[property] = (<any>data)[property];
      }
    }
  }
  requestDate: string;

  init(data?: any) {
    if (data) {
      this.Id = data["Id"];
      this.Code = data["Code"];
      this.Name = data["Name"];
      this.Description = data["Description"];
      this.PhoneNumber = data["PhoneNumber"];
      this.MobileNumber = data["MobileNumber"];
      this.Website = data["Website"];
      this.Email = data["Email"];
      this.OrganizationId = data["OrganizationId"];
      this.ContactPerson = data["ContactPerson"];
      this.Address1 = data["Address1"];
      this.Address2 = data["Address2"];
      this.CountryId = data["CountryId"];
      this.CityId = data["CityId"];
      this.StateId = data["StateId"];
      this.PostalCode = data["PostalCode"];
      this.PrefixId = data["PrefixId"];
      this.Position = data["Position"];
      this.Date = data["Date"];
      this.IsDeleted = data["IsDeleted"];
      this.IsActive = data["IsActive"];
      this.Multisite = data["Multisite"];
      this.LastModifierUserId = data["LastModifierUserId"];
      this.CreatorUserId = data["CreatorUserId"];
      this.RiskId = data["RiskId"];
      this.EacodeId = data["EacodeId"];
      this.NaceCodeId = data["NaceCodeId"];
      this.FormName = data["FormName"];
      // this.File = data["File"];
    }
  }

  static fromJS(data: any): ClientModel {
    data = typeof data === "object" ? data : {};
    let result = new ClientModel();
    result.init(data);
    return result;
  }

  toJSON(data?: any) {
    data = typeof data === "object" ? data : {};
    data["Id"] = this.Id;
    data["Code"] = this.Code;
    data["Name"] = this.Name;
    data["Description"] = this.Description;
    data["PhoneNumber"] = this.PhoneNumber;
    data["MobileNumber"] = this.MobileNumber;
    data["Website"] = this.Website;
    data["Email"] = this.Email;
    data["OrganizationId"] = this.OrganizationId;
    data["ContactPerson"] = this.ContactPerson;
    data["Address1"] = this.Address1;
    data["Address2"] = this.Address2;
    data["CountryId"] = this.CountryId;
    data["CityId"] = this.CityId;
    data["StateId"] = this.StateId;
    data["PostalCode"] = this.PostalCode;
    data["PrefixId"] = this.PrefixId;
    data["Position"] = this.Position;
    data["Date"] = this.Date;
    data["IsDeleted"] = this.IsDeleted;
    data["IsActive"] = this.IsActive;
    data["Multisite"] = this.Multisite;
    data["LastModifierUserId"] = this.LastModifierUserId;
    data["CreatorUserId"] = this.CreatorUserId;
    data["RiskId"] = this.RiskId;
    data["EacodeId"] = this.EacodeId;
    data["NaceCodeId"] = this.NaceCodeId;
    data["FormName"] = this.FormName;
    // data["File"] = this.File;

    return data;
  }

  clone(): ClientModel {
    const json = this.toJSON();
    let result = new ClientModel();
    result.init(json);
    return result;
  }
}

export interface IClientModel {
  Id: number | undefined;
  Code: string | undefined;
  Name: string | undefined;
  Description: string | undefined;
  PhoneNumber: string | undefined;
  MobileNumber: string | undefined;
  Website: string | undefined;

  Email: string | undefined;
  OrganizationId: number | undefined;
  ContactPerson: string | undefined;
  // PersonContactNumber : string | undefined;
  Address1: string | undefined;
  Address2: string | undefined;
  CountryId: number | undefined;
  CityId: number | undefined;
  StateId: number | undefined;
  PostalCode: string | undefined;
  PrefixId: number | undefined;
  Position: string | undefined;
  Date: string | undefined;
  IsDeleted: boolean | undefined;
  IsActive: boolean | undefined;
  Multisite: boolean | undefined;
  LastModifierUserId: number | undefined;
  CreatorUserId: number | undefined;
  RiskId: number | undefined;
  EacodeId: number | undefined;
  NaceCodeId: number | undefined;
  FormName: string | undefined;
  // File: File | undefined;
}

File: Ozone.WebApi/Ozone.Application/DTOs/Setup/ClientModel.cs
using Microsoft.AspNetCore.Http;
using System;
using System.Collections.Generic;
using System.Text;

namespace Ozone.Application.DTOs
{
   public class ClientModel
    {
        public long Id { get; set; }
       
        public string Code { get; set; }
      
        public string Name { get; set; }
       
        public string Description { get; set; }
     
        public string PhoneNumber { get; set; }
      
        public string MobileNumber { get; set; }
       
        public string Website { get; set; }
      
        public string Email { get; set; }
        public long? OrganizationId { get; set; }
       
        public string ContactPerson { get; set; }
       
        public string PersonContactNumber { get; set; }
     
        public string Address1 { get; set; }
       
        public string Address2 { get; set; }
        public long? CountryId { get; set; }
        public string CountryName { get; set; }
        public long? StateId { get; set; }
        public string StateName { get; set; }
        public long? CityId { get; set; }
        public string CityName { get; set; }
        public string PostalCode { get; set; }
        public long? PrefixId { get; set; }
     
        public string Position { get; set; }
        public bool? IsActive { get; set; }
        public bool? Multisite { get; set; }
        public DateTime? CreationTime { get; set; }
        public long CreatorUserId { get; set; }
        public DateTime? LastModificationTime { get; set; }
        public long? LastModifierUserId { get; set; }
        public long? IsDeleted { get; set; }
        public long? DeleterUserId { get; set; }
        public DateTime? DeletionTime { get; set; }
        public DateTime? Date { get; set; }
        public bool? IsProjectExist { get; set; }
        public string SiteName { get; set; }
        public string SiteAddress { get; set; }
        public string SiteCity { get; set; }
        public string SiteCountry { get; set; }
        public int? SiteCount { get; set; }
        public long? OverAllEmployees { get; set; }
        public string OrganizationName { get; set; }
        public string OrganizationCode { get; set; }

        public long? RiskId { get; set; }
        public string RiskName { get; set; }

        public long? EacodeId { get; set; }   
        public string Eacodename { get; set; }
        public long? NaceCodeId { get; set; }
        public string NaceCodeName { get; set; }

        public string FormName { get; set; }

        public IFormFile File { get; set; }

        public string FilePath { get; set; }
        public string ContentType { get; set; }


        //public string SiteContractName { get; set; }
        //public string SiteContractNumber { get; set; }
        //public string SiteEmail { get; set; }
        //public string RequestedService { get; set; }


    }
}
